VirtoCommerce.Domain/Customer/Model/Contact.cs

```diff
@@ -27,8 +28,24 @@ public class Contact : Member, IHasSecurityAccounts
         /// <summary>
         /// All security accounts associated with this contact
         /// </summary>
-        public ICollection<ApplicationUserExtended> SecurityAccounts { get; } = new List<ApplicationUserExtended>();
+        public ICollection<ApplicationUserExtended> SecurityAccounts { get; private set; } = new List<ApplicationUserExtended>();
 
         #endregion
+
+        public virtual object Clone()
+        {
+            var clone = (Contact) MemberwiseClone();
+
+            if (Organizations != null)
+            {
+                clone.Organizations = new List<string>(Organizations);
```

Comment:
We need to make Organizations list copy, 
<code>new List<string>(Organizations)</code>
creates list referencing the same strings.
.ToList() could help.

Reply:
Cloning strings isn't really necessary is it? https://referencesource.microsoft.com/#mscorlib/system/string.cs

Reply:
OK, let's leave it that way.

Reply:
Actually `.ToList()` is much better. It is shorter and it alows to remove `
if (Organizations != null)`:
```csharp
clone.Organizations = Organizations?.ToList();
```

Reply:
Has already been refactored: https://github.com/VirtoCommerce/vc-module-core/pull/111/commits/4a76c3ff26a45968f1b4c3aa9acd1b3332f461b0